LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Ranie Jade Ramiso
+Copyright (c) 2020 godot-kotlin Maintainers and Contributor
```

Comment:
Very picky but it's a license file ^^ "Contributors" i think it should be plural?